data/db-schema/src/main/resources/net/corda/db/schema/rbac/migration/rbac-creation-v5.1.xml

```diff
@@ -0,0 +1,300 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.18.xsd">
+
+    <changeSet author="R3.Corda" id="rbac-creation-v1.0">
+        <createTable tableName="rbac_user">
```

Comment:
I think we should use LBs [modify-data-type](https://docs.liquibase.com/change-types/modify-data-type.html) in this new changeset for 5.1.